performant-software/neatline-spa
# replace OpenLayers2 with Leaflet, enable inclusion in Omeka S module

### What does this PR do?
- Removes OpenLayers2 dependencies and adds leaflet-react for map rendering
- Removes service-worker script and checks in build directory, enabling this project to be included as a submodule and rendered in the Neatline Omeka S module
- Loads public exhibits from the expected API endpoint and provides a "Public View" button to display the exhibit's title alongside a Leaflet map.

### What issues does it address?
- Closes #1 
- Closes #9 

### How to test
The staging instance at https://neatline-3-staging.herokuapp.com/neatline should display a simple table of demo exhibits that have been created through the API. Click the "Public View" button beneath an exhibit's title — this should load another view showing the exhibit title over an embedded Leaflet map centered on London. Clicking "Back to Exhibits" should return you to the initial table.
